pkg/operation/botanist/component/kubeapiserver/dependency_watchdog_test.go

```diff
@@ -67,13 +68,45 @@ var _ = Describe("DependencyWatchdog", func() {
 					Internal:      &scalerapi.ProbeDetails{KubeconfigSecretName: "dependency-watchdog-internal-probe"},
 					PeriodSeconds: pointer.Int32(30),
 				},
-				DependantScales: []*scalerapi.DependantScaleDetails{{
-					ScaleRef: autoscalingv1.CrossVersionObjectReference{
-						APIVersion: "apps/v1",
-						Kind:       "Deployment",
-						Name:       "kube-controller-manager",
+				DependantScales: []*scalerapi.DependantScaleDetails{
+					{
+						ScaleRef: autoscalingv1.CrossVersionObjectReference{
+							APIVersion: "apps/v1",
+							Kind:       "Deployment",
+							Name:       "kube-controller-manager",
+						},
+						ScaleUpDelaySeconds: pointer.Int32(120),
+					},
+					{
+						ScaleRef: autoscalingv1.CrossVersionObjectReference{
+							APIVersion: "apps/v1",
+							Kind:       "Deployment",
+							Name:       "machine-controller-manager",
+						},
+						ScaleUpDelaySeconds: pointer.Int32(60),
+						ScaleRefDependsOn: []autoscalingv1.CrossVersionObjectReference{
+							{
+								APIVersion: appsv1.SchemeGroupVersion.String(),
+								Kind:       "Deployment",
+								Name:       "kube-controller-manager",
+							},
+						},
 					},
-				}},
+					{
+						ScaleRef: autoscalingv1.CrossVersionObjectReference{
+							APIVersion: "apps/v1",
+							Kind:       "Deployment",
+							Name:       "cluster-autoscaler",
+						},
+						ScaleRefDependsOn: []autoscalingv1.CrossVersionObjectReference{
+							{
+								APIVersion: appsv1.SchemeGroupVersion.String(),
```

Comment:
Nit: for consistency reasons with the rest of the similar usages in this file.

```suggestion
								APIVersion: "apps/v1",
```

Reply:
Thanks for pointing out. Resolved with https://github.com/gardener/gardener/commit/94086ac82bc5d5237c701c0e9563d7affdb3fcb4